QueryManager/src/main/java/eu/neclab/ngsildbroker/queryhandler/controller/QueryController.java

```diff
@@ -83,32 +80,53 @@ public Uni<RestResponse<Object>> getEntity(HttpServerRequest request, @QueryPara
 			@QueryParam(value = "localOnly") boolean localOnly, @PathParam("entityId") String entityId,
 			@QueryParam(value = "doNotCompact") boolean doNotCompact,
 			@QueryParam("containedBy") @DefaultValue("") String containedBy, @QueryParam("join") String join,
-			@QueryParam("idsOnly") boolean idsOnly, @QueryParam("joinLevel") @DefaultValue("1") int joinLevel) {
+			@QueryParam("idsOnly") boolean idsOnly, @QueryParam("joinLevel") @DefaultValue("1") int joinLevel,
+			@QueryParam("pick") String pick,@QueryParam("omit") String omit) {
```

Comment:
Pick and Omit parameter should be List of String.

Reply:
done